docs/stubs.rst

```diff
@@ -119,7 +111,9 @@ Two kinds of structured comments are accepted:
   declaring that the variable has type ``X``. However, PEP 526-style [#pep526]_
   variable annotations are preferred over type comments.
 * A ``# type: ignore`` comment at the end of any line, which suppresses all type
-  errors in that line.
+  errors in that line. The type checker mypy supports suppressing certain
```

Comment:
If I recall correctly some other type checkers support similar constructs. A comparison table could be a useful part of the typing docs.